just-mode.el

```diff
@@ -173,6 +173,12 @@ Argument N number of untabs to perform"
   (when (boundp 'evil-shift-width)
     (setq-local evil-shift-width 4))
 
+  ;; Imenu
+  (setq-local imenu-generic-expression
+              '(("setting" "^set +\\([A-Z_a-z][0-9A-Z_a-z-]+\\)\\(?:\\'\\|$\\| \\|:=\\)" 1)
```

Comment:
I think the second `+` here should be a `*` to work on single-character names. Similarly below.